code/modules/mob/living/carbon/carbon.dm

```diff
@@ -595,9 +595,6 @@
 	if(E)
 		. += E.tint
 
-	else
-		. += INFINITY
```

Comment:
?

Reply:
I think this was a fix to a weird / specific bug, probably fixed in a better way by now, surely.